doc/book/box/stream.rst

```diff
@@ -0,0 +1,58 @@
+..  _box_stream:
+
+Streams and interactive transactions
+====================================
+
+Overview
+--------
+
+Since :tarantool-release:`2.10.0-beta1`, streams and interactive transactions
+in them are implemented in iproto.
+
+..  glossary::
+
+    Stream
+        A stream is a concept that supports multiplexing several transactions over one connection.
+        It is a more general approach that allows the implementation
```

Comment:
I would add to this the information that all requests in the stream are executed strictly sequentially